index.js

```diff
@@ -102,7 +102,7 @@ var nativeTypes = {
   Function: function (value) { return typeof value === 'function' },
   Null: function (value) { return value === undefined || value === null },
   Number: function (value) { return typeof value === 'number' },
-  Object: function (value) { return typeof value === 'object' },
+  Object: function (value) { return !!value && typeof value === 'object' },
```

Comment:
Doesn't change my thoughts on it,  but,  semantically it should be the exact same. :)